### PR TITLE
Added function for downloading data set

### PR DESCRIPTION
**Pull request by adam-broussard**
Resolves issue #1 !  Yay!

### CODE REVIEW

File: dataset_ops.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    """
+    Creates a data directory if it doesn't exist and downloads the dataset
+    to that directory.
+
+    Args:
+        savedir (str): The directory where the data will be saved
+    """
+
+    if not os.path.isdir(savedir):
+        os.makedirs(savedir)
```
Comment (youngant) [inline dataset_ops.py:20]:
```suggestion
    os.makedirs(savedir, exist_ok=True)
```

Comment (youngant) [inline dataset_ops.py:20]:
```suggestion
    os.makedirs(savedir, exist_ok=True)
```

File: dataset_ops.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+def download_dataset(savedir='data/'):
+    """
+    Creates a data directory if it doesn't exist and downloads the dataset
+    to that directory.
+
+    Args:
+        savedir (str): The directory where the data will be saved
+    """
+
```
Comment (youngant) [inline dataset_ops.py:18]:
```suggestion

    print('Downloading data csv...', end='')
```

Comment (youngant) [inline dataset_ops.py:18]:
```suggestion
    print('Downloading data csv...', end='')
```

File: dataset_ops.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    if not os.path.isdir(savedir):
+        os.makedirs(savedir)
+
+    kaggle.api.authenticate()
+    kaggle.api.dataset_download_files('austinreese/goodreads-books',
+                                      path=savedir, unzip=True)
```
Comment (youngant) [inline dataset_ops.py:24]:
```suggestion
                                      path=savedir, unzip=True)
    print('done.')
```

Comment (youngant) [inline dataset_ops.py:24]:
```suggestion
                                      path=savedir, unzip=True)
    print('done.')
```

**Review by youngant — CHANGES_REQUESTED** [review]
Suggested some small changes to the data downloader, partially just for practice with the pull request.

File: dataset_ops.py (hunk 2 above)
Comment (adam-broussard) [reply dataset_ops.py:18]: I think this one is a repeat of above, so I'll resolve it

File: dataset_ops.py (hunk 1 above)
Comment (adam-broussard) [reply dataset_ops.py:20]: This looks like another repeat above so I'll delete it as well...

File: dataset_ops.py (hunk 3 above)
Comment (adam-broussard) [reply dataset_ops.py:24]: This looks identical to above, so I'll resolve it

**Review by youngant — APPROVED** [review]
Not sure how I made the duplicates, but all looks good now.

**Pull request merged**